cmd/yggctl/actions.go

```diff
@@ -147,12 +147,15 @@ func messageJournalAction(ctx *cli.Context) error {
 		fmt.Println(compiledTextTemplate.String())
 	case "table":
 		writer := tabwriter.NewWriter(os.Stdout, 4, 4, 2, ' ', 0)
-		fmt.Fprint(
+		_, err := fmt.Fprint(
 			writer,
 			"MESSAGE #\tMESSAGE ID\tSENT\tWORKER NAME\tRESPONSE TO\tWORKER EVENT\tWORKER DATA\n",
 		)
+		if err != nil {
```

Comment:
```suggestion
		header := "MESSAGE #\tMESSAGE ID\tSENT\tWORKER NAME\tRESPONSE TO\tWORKER EVENT\tWORKER DATA\n"
		if _, err := fmt.Fprint(writer, header); err != nil {
```

Reply:
No, I do not want to mix fixing issue and doing refactoring of code.